test/test_common/simulated_time_system_test.cc

```diff
@@ -35,7 +35,7 @@ class SimulatedTimeSystemTest : public testing::Test {
   }
 
   void sleepMsAndLoop(int64_t delay_ms) {
```

Comment:
sleepMsAndLoop calling advanceTime and not sleep seems wrong.

Reply:
agreed; I should rename.